ts/DataGrid/DataGrid.ts

```diff
@@ -268,6 +268,8 @@ class DataGrid {
         this.options = merge(DataGrid.defaultOptions, options);
 
         this.gridContainer.style.height = this.getDataGridSize() + 'px';
+        this.gridContainer.role = 'figure';
+        this.gridContainer.tabIndex = 0;
```

Comment:
We should avoid tabIndex AFAIK, @marvin19 could you confirm?

Reply:
Can be removed